ibis/backends/tests/test_vectorized_udf.py

```diff
@@ -336,6 +339,45 @@ def test_elementwise_udf_overwrite_destruct_and_assign(backend, alltypes):
     backend.assert_frame_equal(result, expected, check_like=True)
 
 
+@pytest.mark.only_on_backends(['pandas', 'pyspark'])
+# TODO - udf - #2553
+@pytest.mark.xfail_backends(['dask'])
+@pytest.mark.xfail_unsupported
+def test_elementwise_udf_destruct_exact_once(backend, alltypes):
+    # xfail on Spark version < 3.1.1
+    try:
+        from distutils.version import LooseVersion
+
+        import pyspark
+
+        if LooseVersion(pyspark.__version__) < LooseVersion("3.1.1"):
```

Comment:
hmm not sure i love this here

can you create a pytest decorator and use it here, similar to this: https://github.com/pandas-dev/pandas/blob/master/pandas/util/_test_decorators.py#L200 (you can use skipif, but have to do the import too)

Reply:
Ha let me try

Reply:
Added marker `min_spark_version`